exec/jdbc-all/pom.xml

```diff
@@ -473,6 +473,8 @@
                <exclude>org/yaml/**</exclude>
                <exclude>hello/**</exclude>
                <exclude>webapps/**</exclude>
+               <exclude>**/org/apache/calcite/avatica/metrics/**</exclude>
+               <exclude>**/org/apache/calcite/avatica/org/**</exclude>
```

Comment:
After removing these exclusion for apache profile what is the size of jdbc-all jar ? based on final size we should reduce the allowed maxSize from ~35MB to that value.

Reply:
The Apache jar size has shrunk by ~1MB from over 34MB to 33393541 bytes; I guess we can decrease the limit from 35MB to 34MB.

Reply:
Would be great to make that change. Other than that, changes looks good to me

Reply:
done!

Reply:
The problem with this PR is that running the test-suite is not enough; as @parthchandra indicated We need to setup few tools and run metadata / DDL & DML queries (involving complex data) to ensure there are no regressions. I am afraid this kind of optimizations will not be easy to include unless we automate the JDBC testing process.